### DISCUSSION TITLE
Hide version number of the lib

### DISCUSSION THREAD

**Discussion by DrGkill** [mozilla/pdf.js#18868] [General]
Dear,

Is there any solution to hide the version number of the lib in the JS console ?

Thanks for your help.

**Comment by Snuffleupagus** [comment]
I'm guessing, since this is a little short on the details, that you're referring to https://github.com/mozilla/pdf.js/blob/81cf42df470f85eb60150bdffca380e3eee79c08/web/app.js#L1543-L1548
If so, there's no option to hide that since it's generally quite helpful to have easy access to such details when users report bugs.
Hence you'd need to edit the source-code to achieve that.

Finally, please note http://mozilla.github.io/pdf.js/getting_started/#introduction (emphasis mine):
> The viewer is built on the display layer and is the UI for PDF viewer in Firefox and the other browser extensions within the project. It can be a good starting point for building your own viewer. ***However, we do ask if you plan to embed the viewer in your own site, that it not just be an unmodified version. Please re-skin it or build upon it.***
